docs/design/StratisSoftwareDesign.lyx

```diff
@@ -6933,5 +6933,423 @@ Based on looking at the existing available building blocks, the best option
  provides.
 \end_layout
 
+\begin_layout Section
+Encryption implementation details
+\end_layout
+
+\begin_layout Standard
+Stratis uses libcryptsetup to manage FDE (full disk encryption) for each
+ block device added to an encrypted pool.
+\end_layout
+
+\begin_layout Subsection
+On-disk format
+\end_layout
+
+\begin_layout Standard
+Stratis uses the LUKS2 encryption format as the basis for the encryption
+ implementation.
+ The LUKS2 header contains information that can be used to discover devices
+ as a Stratis encrypted device.
+\end_layout
+
+\begin_layout Subsubsection
+LUKS2 token format
+\end_layout
+
+\begin_layout Standard
+LUKS2 tokens contain the necessary information to describe how to unlock
+ the device.
+ No private information is contained in the tokens.
+ Stratis lays down two tokens, one for interacting with the passphrase stored
+ in the kernel keyring and another that contains activation information
+ required by Stratis.
+ Token indexes are statically assigned for each token so that new tokens
```

Comment:
I think that here the accepted nomenclature is "token id", not "toke index" so you should use that instead. 

After this sentence you could add an additional one stating the ids of the two tokens. Then you can leave out specifying the ids in the descriptions of the individual tokens.